### PR TITLE
Registers package with the SunPy template

### PR DESCRIPTION
**Pull request by CyclingNinja**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
-*********
+=========
 sunraster
-*********
+=========
+
+sunraster is an open-source Python library that provides the tools to read in and analyze spectrogram data.
+-----------------------------------------------------------------------------------------------------------
```
Comment (Cadair) [inline README.rst:6]:
```suggestion
```

File: docs/conf.py [hunk 2]
```diff
@@ -72,27 +83,23 @@
...
 # -- Options for HTML output -------------------------------------------------
+
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-
-try:
-    from sunpy_sphinx_theme.conf import *
-except ImportError:
-    html_theme = "default"
+html_theme = "alabaster"
```
Comment (Cadair) [inline docs/conf.py:89]:
```suggestion
html_theme = "sunpy"
```

File: docs/conf.py [hunk 3]
```diff
@@ -72,27 +83,23 @@
...
-graphviz_output_format = "svg"
-
-graphviz_dot_args = [
-    "-Nfontsize=10",
-    "-Nfontname=Helvetica Neue, Helvetica, Arial, sans-serif",
-    "-Efontsize=10",
-    "-Efontname=Helvetica Neue, Helvetica, Arial, sans-serif",
-    "-Gfontsize=10",
-    "-Gfontname=Helvetica Neue, Helvetica, Arial, sans-serif",
-]
```
Comment (Cadair) [inline docs/conf.py:98]: This should stay. It should also find it's way into the template.

File: tox.ini [hunk 4]
```diff
@@ -1,53 +1,87 @@
 [tox]
 min_version = 4.0
+requires =
+    tox-pypi-filter>=0.14
 envlist =
-    py{310,311,312}{,-oldestdeps,-devdeps,-online}
-    build_docs
+    py{310,311,312}
```
Comment (Cadair) [inline tox.ini:6]:
```suggestion
    py{310,311,312}{,-online}
```

**Review by Cadair — APPROVED** [review]
lgtm

**Review by DanRyanIrish — APPROVED** [review]

**Pull request merged**